### PR TITLE
beta issue solved

### PR DESCRIPTION
**Pull request by shalinee-walkover**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

File: apps/web-giddh/src/app/purchase/create-purchase-order/create-purchase-order.component.html [hunk 1]
```diff
@@ -862,11 +862,10 @@
                                 <div [collapse]="isOthrDtlCollapsed" class="clearfix max-600 pdL3 mr0">
                                     <div class="row">
                                         <div class="form-group col-xs-4 col-sm-4 mb-0">
-                                            <input type="text" #shippingDate="ngModel" placeholder="Ship Date"
-                                                bsDatepicker name="shippingDate"
-                                                [(ngModel)]="purchaseOrder.templateDetails.other.shippingDate"
-                                                class="form-control" [bsConfig]="{ dateInputFormat: giddhDateFormat }"
-                                                [class.invalid-inp]="shippingDate.errors && (shippingDate.dirty || shippingDate.touched)" />
+                                                <giddh-datepicker #shippingDate="ngModel" placeholder="Ship Date"  [placeholder]="templatePlaceholder?.shippedDateLabel" name="shippingDate"
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/purchase/create-purchase-order/create-purchase-order.component.html:865]: 2 placeholders are used. remove 1 of them.

File: apps/web-giddh/src/app/purchase/create-purchase-order/create-purchase-order.component.html [hunk 2]
```diff
@@ -1505,7 +1504,10 @@
                         <div class="row">
 
                             <div class="form-group col-xs-4">
-                                <input type="text" #shippingDate="ngModel" placeholder="Ship Date" bsDatepicker name="shippingDate" [(ngModel)]="purchaseOrder.templateDetails.other.shippingDate" class="form-control" [bsConfig]="{ dateInputFormat: giddhDateFormat }" [class.invalid-inp]="shippingDate.errors && (shippingDate.dirty || shippingDate.touched)" />
+                                <giddh-datepicker #shippingDate="ngModel" placeholder="Ship Date" [placeholder]="templatePlaceholder?.shippedDateLabel"
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/purchase/create-purchase-order/create-purchase-order.component.html:1507]: 2 placeholders are used. remove 1 of them.

**Review by ravinderwalkover — CHANGES_REQUESTED** [review]

File: apps/web-giddh/src/app/purchase/create-purchase-order/create-purchase-order.component.html (hunk 1 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/purchase/create-purchase-order/create-purchase-order.component.html:865]: which one should I remove ?

File: apps/web-giddh/src/app/purchase/create-purchase-order/create-purchase-order.component.html (hunk 2 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/purchase/create-purchase-order/create-purchase-order.component.html:1507]: which one should I remove ?

File: apps/web-giddh/src/app/purchase/create-purchase-order/create-purchase-order.component.html [hunk 3]
```diff
@@ -1504,8 +1504,7 @@
                         <div class="row">
 
                             <div class="form-group col-xs-4">
-                                <giddh-datepicker #shippingDate="ngModel" placeholder="Ship Date" [placeholder]="templatePlaceholder?.shippedDateLabel"
-                                    name="shippingDate" [(ngModel)]="purchaseOrder.templateDetails.other.shippingDate"
+                                <giddh-datepicker #shippingDate="ngModel" placeholder="Ship Date"                                    name="shippingDate" [(ngModel)]="purchaseOrder.templateDetails.other.shippingDate"
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/purchase/create-purchase-order/create-purchase-order.component.html]: remove space

**Review by ravinderwalkover — APPROVED** [review]

**Pull request merged**